packages/farcaster-js-hub-rest/package.json

```diff
@@ -12,13 +12,13 @@
   "author": "Gavi Galloway",
   "license": "MIT",
   "dependencies": {
-    "@farcaster/core": "^0.13.1"
+    "@farcaster/core": "^0.13.1",
+    "ethers": "^6.8.1"
```

Comment:
This is a big dependency to introduce. Double-checking that we want/need to. 

I see it used for importing `Wallet` as part of the `eip712SignerFromPhrase` function. Is there another approach that wouldn't pull in ethers? 

I'm cool if the answer is "no," and don't spend a bunch of time on it. Just something to think about, because ethers is both large and finicky if people are already importing a different version of it into their project.

Reply:
updated to replace ethers with viem